examples/vision/md/adamatch.md

```diff
@@ -26,9 +26,16 @@ be installed using the following command:
 
 
 ```python
-!pip install -q tf-models-official
+!pip install -q tf-models-official==2.9.2
 ```
 
+<div class="k-default-codeblock">
+```
+[33mWARNING: You are using pip version 21.3.1; however, version 22.1.2 is available.
+You should consider upgrading via the '/home/johannes/Projects/keras-io/venv/bin/python -m pip install --upgrade pip' command.[0m
```

Comment:
Please remove this block

Reply:
Sorry for the late reply on this PR as well, I have now updated the code and removed the block